### ISSUE TITLE
We need a Desktop version

### ISSUE THREAD

**Issue by 4e576rt8uh9ij9okp** [futo-org/grayjay-android#743]
Hi everyone and I know, some people might have talked about this so will I, just because we actually need a Desktop version.

**80% of my GrayJay use:**
So I have GrayJay on my /e/OS Google Pixel, I very like it, there were features added 2 days after I said "this would be nice", I have a Subscription Filter that shows me all my Podcasts that I'm listening to at work.

**The issue**
Currently my life looks like I need to duplicate every action that I do on GrayJay and Youtube and vice versa.
So If I sub to someone on GrayJay... I sub to that person on Youtube because as every normal person I own also a Computer and this back and forth subbing is very annoying.

**The fix**
Make a desktop version of GrayJay. I know it's a big request but this would make me go from Youtube fully to GrayJay.
I think a Electron App of GrayJay wouldn't be a bad thing, except I have no clue how GrayJay works.

**Comment by EnzDev** [comment]
We all want a desktop version ! But you should note some important keys facts and workarounds. (I'm just a user, not affiliated with the project in any way)

1. [Grayjay FAQ](https://grayjay.app/#faq) says: 
> **Do you have a desktop version?**
> A desktop version is being considered in the future. However, our focus right now is the Android app.

2. The app [have a big focus on keeping your data on your device](https://grayjay.app/#privacy) so the 'sync' problem would stay the same, you would have to backup/restore your saves to synchronize your Subscription/Playlists and Watch History.

3. Finally, the most interesting thing for you I think :smirk: FUTO has created the [FCast](https://gitlab.futo.org/videostreaming/fcast) protocol, supported in Grayjay. They provides  [_receiver_ apps](https://gitlab.futo.org/videostreaming/fcast/-/tree/master/receivers/electron?ref_type=heads) for all platforms [(link to the downloads)](https://gitlab.com/futo-org/fcast/-/releases) so you can cast any video easily from Grayjay to your personal computer.

**Comment by armilancode** [comment]
Louis said the desktop app is coming very soon, so stay tuned.

**Comment by 4e576rt8uh9ij9okp** [comment]
About the **sync** part, it doesn't need to be on FUTO's servers, how about creating a way for me to setup my own mini server somewhere, could be a end-to-end encrypted VPS or even a self hosted Raspberry Pi4.
I would love the idea that I could just paste my IP and a key and then just sync my data from my own apps. The files are still MINE.

**Comment by birdup000** [comment]
I think Electron is clunky for the dev team reading this feature request I recommend using Tauri.

**Comment by 4e576rt8uh9ij9okp** [comment]
> I think Electron is clunky I recommend using Tauri since it's more secure overall and its faster.

Yes because Electron is just a minimalistic Browser with NodeJS.
Also It's just crazy how bloated Electron is... https://www.levminer.com/blog/tauri-vs-electron
Nice idea but the execution is so bad...

**Comment by Pandapip1** [comment]
For Linux users, Waydroid is an option and for Windows users, Windows subsystem for Android might be of interest.

**Comment by Phexyaa** [comment]
I'd like to suggest a more cross platform technology like Blazor or Maui. That would allow you to write the application once and have it available to all platforms via browers. Maybe the Android app could be re-written in C# which could then be run in a browser using Blazor

**Comment by paaspaas00** [comment]
They're working on a desktop version, apparently it will begin to be rolled out (dunno if as a beta or alpha) in the next 1-1½ months
